posthog/api/prompt.py

```diff
@@ -34,20 +34,22 @@ def my_prompts(self, request: request.Request, **kwargs):
             )
             local_states.append(parsed_state)
 
+        team_id = 2  # we are testing prompts created by us, on PostHog Cloud
```

Comment:
This constant kinda looks like a magic number, could be a bit clearer by naming it `POSTHOG_CLOUD_PRODUCTION_TEAM_ID` or something like that